spec/models/item_order_spec.rb

```diff
@@ -1,5 +1,24 @@
 require "rails_helper"
 
 RSpec.describe ItemOrder, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'is valid' do
+    it 'has a default quantity' do
+      item_order = create(:item_order)
+      expect(item_order.quantity).to eq(1)
+    end
+
+    it 'can be saved' do
+      item_order = build(:item_order)
+      item_order.save
+      expect(item_order.id).to be_truthy
+    end
+  end
+
+  describe 'it is invalid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.